bin/tact.mjs

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+import * as main from "../dist/node.js";
+import meow from "meow";
+
+const cli = meow(
+  `
+Usage
+  $ tact --config <config_path>
+
+Options
+  --project <project_name>, -r <project_name>  
+  --version                                    Prints the current tact version
+  --func                                       Outputs the intermediate FunC code.
+  --check                                      Performs type checking and stops.
+  --help                                       Displays this text
+
+Examples
+  $ tact --version
+  1.2.0
+`,
+  {
+    importMeta: import.meta,
+    flags: {
+      config: { shortFlags: "c", type: "string", isRequired: true },
+      project: { shortFlag: "p", type: "string", isMultiple: true },
+      version: { shortflag: "v", type: "boolean" },
+      check: { shortflag: "c", type: "boolean" },
+      func: { shortFlag: "f", type: "boolean" },
```

Comment:
Conflicting short versions for `--config` and `--check`. I think `--check` and `--func` should not have short versions as I wouldn't expect those to be used very often by users